Makefile

```diff
@@ -21,22 +21,32 @@ BUILD_TIME=$(shell date +%s)
 GIT_COMMIT=$(shell git rev-parse HEAD)
 VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
 
+.PHONY: all
+all: audit test build
+
+.PHONY: audit
+audit:
+	nancy go.sum
+
+.PHONY: build
 build:
 	@mkdir -p $(BUILD_ARCH)/$(BIN_DIR)
 	go build -o $(BUILD_ARCH)/$(BIN_DIR)/$(MAIN) -ldflags="-X 'main.BuildTime=$(BUILD_TIME)' -X 'main.GitCommit=$(GIT_COMMIT)' -X 'main.Version=$(VERSION)'"
 
+.PHONY: debug
 debug: build
 	HUMAN_LOG=1 VAULT_TOKEN=$(APP_TOKEN) VAULT_ADDR=$(VAULT_ADDR) go run -ldflags="-X 'main.BuildTime=$(BUILD_TIME)' -X 'main.GitCommit=$(GIT_COMMIT)' -X 'main.Version=$(VERSION)'" -race main.go
 
+.PHONY: acceptance
 acceptance:
 	HUMAN_LOG=1 VAULT_TOKEN=$(APP_TOKEN) VAULT_ADDR=$(VAULT_ADDR) go run main.go
 
+.PHONY: test
 test:
 	go test -cover $(shell go list ./... | grep -v /vendor/)
 
+.PHONY: vault
 vault:
 	@echo "$(VAULT_POLICY)"
 	@echo "$(TOKEN_INFO)"
 	@echo "$(APP_TOKEN)"
-
-.PHONY: vault debug build test
```

Comment:
Are you sure it is okay to remove line 42? In some of the other services where the dependency auditing has been added, this line is not removed.

Reply:
It is fine as Jon has explicitly set the phony targets above each command, so it is okay.

Reason for setting phony targets - https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_node/make_34.html